status-whisper-usage-spec.md

```diff
@@ -59,6 +60,14 @@ Whisper's Proof Of Work algorithm is used to deter denial of service and various
 * proof-of-work requirement not larger than `0.002`
 * time-to-live not lower than `10` (in seconds)
 
+## Rate limiting
+
+In order to provide a very basic Denial-of-Service attack protection, there SHOULD be a rate limiting applied to leaf Whisper nodes, i.e. nodes not being controler within a cluster.
+
+Currently, the rate limiting is applied on peer IDs and IPs from which the packets come from. It is still in progress to decide on specific numbers. The Whisper nodes within a cluster SHOULD be whitelisted to avoid dropping connection when broadcasting messages between each other. When a limit is hit, the packet or the connection MUST be dropped (TBD).
```

Comment:
Not sure about the use of "Currently" "in the future/TBD" - for temporal WIP stuff lets put in comment and only describe _what is_ in the spec

Reply:
That's why it is a draft PR. I haven't finalized the numbers. When I do, we can be more specific. However, I don't even know whether we should put any specific numbers... It depends on a situation.